cmake/OpenCVModule.cmake

```diff
@@ -499,6 +486,21 @@ function(__ocv_resolve_dependencies)
     endforeach()
   endif()
 
+  # add reverse wrapper dependencies (BINDINDS)
+  foreach(the_module ${OPENCV_MODULES_BUILD})
```

Comment:
Is it possible that `OPENCV_MODULES_BUILD` would change in the next block (lines 508-524) due to missing dependencies?

Reply:
These dependencies are specified as "OPTIONAL", so this case should be handled.